src/wasm/extract.cpp

```diff
@@ -80,10 +87,9 @@ void file_output::close() {
   return;
 }
 
-
 void file_output::settime(time_t t){
-  if(zip_open_)
-    zip_.setTime(zipindex_, t);
+  if(file_open_)
+    zs_entrydatetime(ze_, t);
```

Comment:
Missing space and curly bracket.
```suggestion
  if (file_open_) {
    zs_entrydatetime(ze_, t);
  }
```